src/assets/docker-entrypoint.sh

```diff
@@ -36,6 +36,8 @@ if (echo "${ACTIVEMQ_ARTEMIS_VERSION}" | grep -Eq "(1\\.[^0-2]\\.[0-9]+|2\\.[0-9
     -v "$(hostname)" ../etc/broker.xml
 fi
 
+# add ability to override acceptor options
+sed -i "/brokerConfig.acceptorArtemisArgs/!s/\(.*<acceptor.*\"artemis\".*61616\)\(.*\)\(<\/acceptor>\)/\1\${brokerConfig.acceptorArtemisArgs:\2}\${brokerConfig.acceptorArtemisArgsExtra}\3/g" ../etc/broker.xml
```

Comment:
I'm unsure if this was intentional. It seems like two different features are blended together in one PR.

Reply:
intentional.